src/components/explorer/SearchSummaryModal.js

```diff
@@ -1,224 +1,137 @@
+import React, { useEffect, useState } from "react";
+import { Col, Divider, Modal, Row, Statistic, Typography } from "antd";
 import CustomPieChart from "../overview/CustomPieChart";
 import Histogram from "../overview/Histogram";
-import {SEX_VALUES} from "../../dataTypes/phenopacket";
-import {explorerSearchResultsPropTypesShape} from "../../propTypes";
-import {mapNameValueFields} from "../../utils/mapNameValueFields";
+import { explorerSearchResultsPropTypesShape } from "../../propTypes";
 import { useSelector } from "react-redux";
 
 const CHART_HEIGHT = 300;
 const CHART_ASPECT_RATIO = 1.8;
 const MODAL_WIDTH = 1000;
 
-const SearchSummaryModal = ({searchResults, ...props}) => {
+const serializePieChartData = (data) => Object.entries(data).map(([key, value]) => ({ name: key, value }));
+const serializeBarChartData = (data) => Object.entries(data).map(([key, value]) => ({ ageBin: key, count: value }));
 
-    const otherThresholdPercentage = useSelector((state) => state.explorer.otherThresholdPercentage);
-    const thresholdProportion = otherThresholdPercentage / 100;
-    const searchFormattedResults = searchResults.searchFormattedResults || [];
 
-    // this doesn't work, many searches incorrectly return all experiments instead of a subset
-    // const experiments = searchResults?.results?.results?.experiment || [];
+const SearchSummaryModal = ({ searchResults, ...props }) => {
+    const [isFetching, setIsFetching] = useState(false);
+    const [data, setData] = useState(null);
 
-    // instead pull experiments from phenopackets
-    const experiments = searchFormattedResults.flatMap(r => r.experiments);
+    const katsuUrl = useSelector((state) => state.services.itemsByArtifact.metadata.url);
 
-    const histogramFormat = (ageCounts) => {
-        // only show age 110+ if present
-        if (!ageCounts[110]) {
-            delete ageCounts[110];
-        }
-        return Object.keys(ageCounts).map(age => {
-            return {ageBin: age, count: ageCounts[age]};
-        });
-    };
-
-    const ageBinCounts = {
-        0: 0,
-        10: 0,
-        20: 0,
-        30: 0,
-        40: 0,
-        50: 0,
-        60: 0,
-        70: 0,
-        80: 0,
-        90: 0,
-        100: 0,
-        110: 0,
-    };
-
-    // Individuals summary
-    const numIndividualsBySex = Object.fromEntries(SEX_VALUES.map(v => [v, 0]));
-
-    // - Phenotypic features summary
-    const numPhenoFeatsByType = {};
-
-    // - Individuals' diseases summary - from phenopackets
-    const numDiseasesByTerm = {};
-
-    // Biosamples summary
-    // TODO: More ontology aware
-
-    const numSamplesByTissue = {};
-    const numSamplesByHistologicalDiagnosis = {};
+    useEffect(() => {
+        setIsFetching(true);
 
-    // Experiments summary
-    const numExperimentsByType = {};
+        const myHeaders = new Headers();
+        myHeaders.append("Content-Type", "application/json");
```

Comment:
I know this is moved from another place, but this never changes; give this a better name, move it outside of the effect / component, treat it like a constant (since it is), and initialize it all in one go, like:

```js
const JSON_REQ_HEADERS = new Headers({
    "Content-Type": "application/json",
}); 
```